package.json

```diff
@@ -8,7 +8,8 @@
   },
   "license": "MIT",
   "dependencies": {
-    "ft-next-logger": "^2.2.1",
+    "fetchres": "^1.5.1",
+    "ft-next-logger": "^2.2.3",
```

Comment:
could get the last logger too, v3, has a slight api change - probably just need to change to

```
const logger = require('ft-next-logger').logger;
```